### PR TITLE
Add new package: libuinputplus

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libuinputplus/package.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from spack import *
+
+
+class Libuinputplus(CMakePackage):
+    """A c++ wrapper around libuinput."""
+
+    homepage = "https://github.com/YukiWorkshop/libuInputPlus"
+    url      = "https://github.com/YukiWorkshop/libuInputPlus/archive/v0.1.4.tar.gz"
+
+    version('0.1.4', sha256='a537e156d11ad00c643b93cbd9b712d3ec9d0ae8e40731ff763fe9a6ffe97458')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/libuinputplus/package.py:15]: Can you also add newer versions? It seems there are v0.2.1 and v0.2.0 available.

**Review by alalazo — CHANGES_REQUESTED** [review]
Curious if there is a reason to add only old versions of the package?

File: var/spack/repos/builtin/packages/libuinputplus/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/libuinputplus/package.py:15]: Emmm, there're build errors in `0.2.1` and `0.2.0` in the `install` phase.

Comment (darmac) [reply var/spack/repos/builtin/packages/libuinputplus/package.py:15]: see as:
```
make: *** No rule to make target 'install'.  Stop.
==> Error: ProcessError: Command exited with status 2:
    'make' '-j4' 'install'

1 error found in build log:
     56    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-libuinputplus-0.2.0-7yrqnoun6kbaed7huybjdxp2cyhyhbk5/spack-build-7yrqnou'
     57    [100%] Built target uInputPlus_test
     58    make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-libuinputplus-0.2.0-7yrqnoun6kbaed7huybjdxp2cyhyhbk5/spack-build-7yrqnou'
     59    /home/xiaojun/spack/opt/spack/linux-centos8-aarch64/gcc-8.3.1/cmake-3.19.2-v4kxpjumu6q2sn357hshnf4zmhn4nj45/bin/cmake -E cmake_progress_start /tmp/root/spack-stage/spack-stage-libuinputplus-0.2.0-7yrqnoun6kbaed7huybjdxp2cyhyhbk5/
           spack-build-7yrqnou/CMakeFiles 0
     60    ==> libuinputplus: Executing phase: 'install'
     61    ==> [2021-03-13-10:40:46.229196] 'make' '-j4' 'install'
  >> 62    make: *** No rule to make target 'install'.  Stop.
```

I think the `cmake` is not generate `install` aim.

**Pull request closed without merging**